packages/terra-date-picker/src/clinical-lowlight-theme/DatePicker.module.scss

```
@@ -34,7 +34,7 @@
     --terra-date-picker-input-date-component-focus-background-color: transparent;
     --terra-date-picker-input-date-component-focus-border: 0;
     --terra-date-picker-input-date-component-focus-box-shadow: none;
-    --terra-date-picker-input-date-component-initial-focus-background-color: rgba(139, 194, 249, 0.3);
+    --terra-date-picker-input-date-component-initial-focus-border: 2px dashed #000;
```

```suggestion
    --terra-date-picker-input-date-component-initial-focus-border: 2px dashed #fff;
```
Colour should be white in lowlight theme and not black